### PR TITLE
ENDOC-374 Update the AWS install guide for 6.3.2. Update the k8s vers…

### PR DESCRIPTION
**Pull request by nshaw**
…ion in the azure install guide.

### CODE REVIEW

File: vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md [hunk 1]
```diff
@@ -19,7 +18,6 @@ The steps below walk you though installing the Entando platform in an EKS cluste
 - Configure an IAM role to allow kubernetes to manage the cluster
 - Create an EKS cluster with 5 nodes (to allow expansion for microservices)
 - Install nginx as an ingress controller in the cluster
-- Register a domain (if you don't already have one) and configure it for wildcard subdomains.
 - Install Entando
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md:22]: Would take out the period. The list does not have periods.

File: vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md [hunk 2]
```diff
@@ -32,59 +30,70 @@ These steps will use the AWS console to create the cluster. If you’re already
...
+   - Go to `Services → Elastic Kubernetes Service → Clusters` and click on your cluster name.
+   - Go to `Configuration` and the `Compute` tab
    - Click `Add Node Group`
    - `Name`: give your group a name, e.g. `node-1`
    - `Node IAM Role`: Select the cluster role you created above. If the role doesn't appear, verify that you added the extra policies to the role.
-   - `Subnets` - VPC subnets should already be setup and selected.
-   - Select `Allow remote access to nodes`.  Follow the links to create a new SSH key pair if you don't already have one.
    - Click `Next`
+   - Review the compute and scaling configuration. Typically the AWS defaults should work.
    - AMI type: `Amazon Linux 2`
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md:78]: Unsure but should compute or scaling be capitalized?

File: vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md [hunk 3]
```diff
@@ -107,55 +116,63 @@ ingress-nginx-controller             LoadBalancer   10.100.102.83    ad234bd11a1
...
-```
+```sh
 kubectl create namespace entando
 ```
 3. Install the namespace scoped resources
-```
+```sh
 kubectl apply -n entando -f https://raw.githubusercontent.com/entando/entando-releases/v6.3.2/dist/ge-1-1-6/namespace-scoped-deployment/orig/namespace-resources.yaml
 ```
+If you run `kubectl -n entando get pods` you'll see the Entando operator is now running.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md:138]: think there should be a comma after 'pods'.

File: vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md [hunk 4]
```diff
@@ -107,55 +116,63 @@ ingress-nginx-controller             LoadBalancer   10.100.102.83    ad234bd11a1
...
-   - Add `entando.ingress.class: "nginx"
-4. Run `kubectl apply -f sample-configmaps/entando-operator-config.yaml -n entando`
-5. In `values.yaml` in the root directory set the following value:
+3. Edit `sample-configmaps/entando-operator-config.yaml` and add the following settings in the data section
+```yaml
+  entando.requires.filesystem.group.override: "true"
+  entando.ingress.class: "nginx"
+```  
+4. Now create the config map for the operator
+```sh
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md:156]: should config map always be capitalized or is it ok when it works as common noun? Will add to Style guide.

**Review by jyunmitch — COMMENTED** [review]
did not repeat comments for the same changes in a different file. Maybe should do that?

**Comment by Lyd1aCla1r3** [comment]
The use of periods is supremely inconsistent... Should we be focused on your edits alone or documents as a whole? I want to pull this to VSCode to view the modified files in Vuepress and make changes within my local environment - a more sophisticated collaboration where multiple branches and contributors affect a common file. :) Overkill?

File: vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md:22]: This line was deleted so should be all set.

File: vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md:78]: Done

File: vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md (hunk 3 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md:138]: Yep, done

File: vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md (hunk 4 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md:156]: Done

**Review by jyunmitch — APPROVED** [review]

**Pull request merged**